site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -20,10 +20,9 @@ A macOS computer is required to build the Mattermost iOS mobile app.
 
 Install the following prerequisite software to develop and build the iOS or Android apps. For macOS, we recommend using [Homebrew](https://brew.sh) as a package manager.
 
-#### Install [NodeJS 16](https://nodejs.org/en/).
-This includes NPM 7 which is also needed.
+#### Node and NPM
+For mobile V1, we recommend using npm version 7 (with either Node 15 or 16). For mobile v2 (gekidou), we recommend npm version 8 and Node 16. To make switching easier, many of our team use [nvm](https://github.com/nvm-sh/nvm) to manage the npm and node versions.
```

Comment:
we are not using npm 8 in any of the branches of mobile

Reply:
Oh, I just assumed we were because it's the default when you `nvm install 16` and `nvm use 16` -- nvm pairs npm 8 with node 16. I'll change it.